web/src/widgets.ts

```diff
@@ -0,0 +1,11 @@
+import * as poll_widget from "./poll_widget";
+import * as todo_widget from "./todo_widget";
+import * as widgetize from "./widgetize";
+const zform: any = require("./zform");
```

Comment:
Don’t ever use `any`, as it [turns off type checking](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any).

Before we can migrate `widgets.js` to TypeScript, we need to convert `zform.js`, as well as a long chain of its other dependencies.

Reply:
Hi @andersk 

Thank you for the feedback, appreciate it!🙏 Do you have another module in mind that is ready to be converted?